### PR TITLE
Add hover states for shh and lock icons.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/chat/inbox/row/big-team-channel/index.js [hunk 1]
```diff
@@ -41,6 +49,8 @@ class BigTeamChannel extends PureComponent<Props> {
               styles.channelBackground,
               this.props.isSelected && styles.selectedChannelBackground,
             ])}
+            onMouseLeave={() => this.setState({isHovered: false})}
```
Comment (chrisnojima) [inline shared/chat/inbox/row/big-team-channel/index.js]: should just make a this._onMouseLeave

File: shared/chat/inbox/row/big-team-channel/index.js [hunk 2]
```diff
@@ -25,13 +25,21 @@ type Props = {
...
+
 const HoverBox = isMobile
   ? Box
   : glamorous(Box)({
       ':hover': {backgroundColor: globalColors.blueGrey2},
     })
 
-class BigTeamChannel extends PureComponent<Props> {
+class BigTeamChannel extends PureComponent<Props, State> {
+  state = {
```
Comment (chrisnojima) [inline shared/chat/inbox/row/big-team-channel/index.js]: if we're doing this we likely dont need Hoverbox anymore?

Comment (adamjspooner) [reply shared/chat/inbox/row/big-team-channel/index.js]: Using `state` for the hover state is still buggy. I'd rather continue using `glamorous` until we/I can work out a better solution. This is just to get the icon hovers working. I'm open to suggestions for making it better.

File: shared/chat/inbox/row/filter-big-team-channel/index.js [hunk 3]
```diff
@@ -19,17 +21,41 @@ type Props = {
...
 }
 
-class FilterBigTeamChannel extends PureComponent<Props> {
+type State = {
+  isHovered: boolean,
+}
+
+const HoverBox = isMobile
+  ? Box
+  : glamorous(Box)({
```
Comment (chrisnojima) [inline shared/chat/inbox/row/filter-big-team-channel/index.js]: note we don't need this anymore with this pr (https://github.com/keybase/client/pull/13870). There are generated classNames for hover so you can just add `className='.hover_color_blueGrey2'`

**Comment by adamjspooner** [comment]
@chrisnojima, that class name only changes text color. Should we add the same for background color?

**Comment by adamjspooner** [comment]
@chrisnojima, this is ready for another review.

**Comment by adamjspooner** [comment]
Totally agreed. I added an issue for our next sprint.

**Pull request merged**